docs/prisma2-cli.md

```diff
@@ -36,30 +36,30 @@ The following environment variables can be provided:
 
 #### `prisma2 init`
 
-Sets up Prisma (i.e. Photon and/or Lift) via an interactive wizard.
+Sets up Prisma (i.e., Photon and/or Lift) via an interactive wizard. You can specify your database connection and/or create a new database to work with. For a list with currently supported database engines, see the [Lift documentation](./lift/use-only-lift.md).
```

Comment:
>  For a list with currently supported database engines, see the [Lift documentation](./lift/use-only-lift.md).

Instead of linking to the Lift docs here, I'd suggest to link to this page: https://github.com/prisma/prisma2/blob/master/docs/supported-databases.md (also I'd just call it "databases" instead of "database engines")

Reply:
will be fixed in next commit